python_client/pyrefinebio/sample.py

```diff
@@ -0,0 +1,197 @@
+        offset=None,
+        dataset_id="",
+        experiment_accession_code="",
+        accession_codes="",
+    ):
+        """
+        Returns a list of samples.
+
+        Usage:
+            `import pyrefinebio`
+            `samples = pyrefinebio.Samples.llist()`
+
+        Parameters:
+            ordering (str):  which field to use when ordering the results
+            title (str)
+            organism (str)
+            source_database (str)
+            source_archive_url (str)
+            has_raw (str)
+            platform_name (str)
+            technology (str)
+            manufacturer (str)
+            sex (str)
+            age (number)
+            specimen_part (str)
+            genotype (str)
+            disease (str)
+            disease_stage (str)
+            cell_line (str)
+            treatment (str)
+            race (str)
+            subject (str)
+            compound (str)
+            time (str)
+            is_processed (str)
+            is_public (str)
+            limit (int):                     Number of results to return per page.
+
+            offset (int):                    The initial index from which to return the results.
+
+            dataset_id (str):                Filters the result and only returns samples
+                                             that are added to a dataset.
+
+            experiment_accession_code (str): Filters the result and only returns only
+                                             the samples associated with an experiment
+                                             accession code.
+
+            accession_codes (str):           Provide a list of sample accession codes
+                                             separated by commas and the endpoint will
+                                             only return information about these samples.
+        """
+
+        params = {
+            "ordering": ordering,
+            "title": title,
+            "organism": organism,
+            "source_database": source_database,
+            "source_archive_url": source_archive_url,
+            "has_raw": has_raw,
+            "platform_name": platform_name,
+            "technology": technology,
+            "manufacturer": manufacturer,
+            "sex": sex,
+            "age": age,
+            "specimen_part": specimen_part,
+            "genotype": genotype,
+            "disease": disease,
+            "disease_stage": disease_stage,
+            "cell_line": cell_line,
+            "treatment": treatment,
+            "race": race,
+            "subject": subject,
+            "compound": compound,
+            "time": time,
+            "is_processed": is_processed,
+            "is_public": is_public,
+            "limit": limit,
+            "offset": offset,
+            "dataset_id": dataset_id,
+            "experiment_accession_code": experiment_accession_code,
+            "accession_codes": accession_codes,
+        }
```

Comment:
For this, I don't think we should have the string fields default to `""`. I think we should only post the ones that are passed in. There's an easy way to do this with `def search(**kwargs):` (see https://realpython.com/python-kwargs-and-args/). Then you can just do `get("samples", params=kwargs)`, after you verify that they're passing in valid fields/types.